adapters/oidc/installed/src/main/java/org/keycloak/adapters/installed/KeycloakInstalled.java

```diff
@@ -567,10 +567,9 @@ protected ResteasyClient getResteasyClient() {
     }
 
     protected ResteasyClient createResteasyClient() {
-        return new ResteasyClientBuilder()
+        return ((ResteasyClientBuilder) ResteasyClientBuilder.newBuilder())
                 .connectionCheckoutTimeout(1, TimeUnit.HOURS)
                 .connectionTTL(1, TimeUnit.HOURS)
-                .socketTimeout(1, TimeUnit.HOURS)
                 .disableTrustManager().build();
```

Comment:
I see that the trustmanager is disabled here. I'm lacking a comment why this would be a good idea here, and also doing this as a default without an option to re-configure it. I know this hasn't been changed in this PR, so it might a new issue. 

I had a look into [keycloak-documentation](https://github.com/keycloak/keycloak-documentation/blob/main/securing_apps/topics/oidc/java/installed-adapter.adoc) to find out what this class is trying to achieve, and didn't find a reason why not to check the certificates of the Keycloak server.

Reply:
Yeah, I would deal with it in a separate issue. But I wouldn't worry about that until we have any issue reported by the community.

This adapter in particular is mainly developed by community and limited to very specific use cases.